playwright.config.ts

```diff
@@ -1,39 +1,23 @@
 import { defineConfig, devices } from '@playwright/test';
 
+// eslint-disable-next-line no-restricted-exports
```

Comment:
footgun

Reply:
I can remove it, but then it'll be with that annoying yellow underline
![image](https://github.com/user-attachments/assets/f45c91d5-baa5-4d8f-80cf-f94d19e946a6)

Reply:
i mean our rule is a footgun for such case